android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -54,23 +54,21 @@
 		"upgradesTo": "Swordsman",
 		"attackSound": "nonmetalhit"
 	},
-	/*
 	{
 		"name": "Jaguar",
 		"unitType": "Melee",
-		"uniqueTo": "Aztec",
+		"uniqueTo": "Aztecs",
 		"replaces": "Warrior",
 		"movement": 2,
 		"strength": 8,
 		"cost": 40,
 		"hurryCostModifier": 20,
 		"obsoleteTech": "Metal Casting",
-		"uniques": ["+33% combat bonus in Forest/Jungle","Woodsmah","Heals 25 damage if it kills an Unit"],
+		"uniques": ["+33% combat bonus in Forest/Jungle", "Heals [2] damage if kills an enemy"],
```

Comment:
That can't be right. 2 damage is nothing, tis is probably on the 1-10 scale instead of on the 1-100.
Also, if we're Uniqueifying, "+[amount] Strength in [terrain]"

Reply:
They increased it to 25 in G&K.
 "+[amount] Strength in [terrain]" does not exist yet. Will create

Reply:
Oh it does exist but Mohawk Warrior use deprecated one for some reason.
Changed both Jaguars and Mohawk Warriors to "+[]% combat bonus in []"
Changed Jaguars to heal 25 units, like it did in G&K

Reply:
Vegetation should be separate val I think. 
e.g baseTerrain Plains with feature Hill and vegetation forest

Reply:
![2020-08-29-18:20:42_1920x1080_scrot](https://user-images.githubusercontent.com/69697985/91642469-48df1580-ea34-11ea-9fcd-d7f474a2351b.png)
I am thinking about implementing this next.
"Vegetation" unique will not be needed for this. Ski infantry will not have bonus in forest hill tundra because forest will be top item in TerrainFeatures list. And only top item will be checked in "+[]% combat bonus in []". 
Put back                
```
 // except when there is a vegetation
                (tile.terrainFeature != Constants.forest
                        || tile.terrainFeature != Constants.jungle))
```
for now?